internal/server/methodhandler_test.go

```diff
@@ -100,7 +100,7 @@ func TestDriver_ProcessMethodCall(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			// create device client and open connection
 			endpoint := test.Protocol + test.Address
-			client := opcua.NewClient(endpoint, opcua.SecurityMode(ua.MessageSecurityModeNone))
+			client, _ := opcua.NewClient(endpoint, opcua.SecurityMode(ua.MessageSecurityModeNone))
```

Comment:
Any chance this could return an error? I guess it depends on if the python server failed to start.

Reply:
Maybe, but in either case the test would fail. I guess I could change it to be more specific about what failed.